perllib/FixMyStreet/App/Form/Noise.pm

```diff
@@ -303,23 +295,23 @@ has_page source_known_address => (
         if ($form->previous_form) {
             $options = $form->previous_form->addresses;
         } else {
-            # Need to perform same lookup as UPRN form...
-            $options = [
-                { value => 'house1', label => 'House 1' },
-                { value => 'house2', label => 'House 2' },
-                { value => 'house3', label => 'House 3' },
-                { value => 'missing', label => 'I can’t find the address' },
-            ];
+            my $saved_data = $form->saved_data;
+            $options = $form->c->cobrand->addresses_for_postcode($saved_data->{source_postcode});
+            push @$options, { value => 'missing', label => 'I can’t find my address' };
```

Comment:
I notice if they select this option then the confirmation form at the end has "missing" as their address rather than what they put in the box on the What is your Address page.